contribs/gmf/src/services/permalink.js

```diff
@@ -692,6 +704,11 @@ gmf.Permalink.prototype.registerLayer_ = function(layer, opt_init) {
  * @private
  */
 gmf.Permalink.prototype.unregisterLayer_ = function(layer) {
+
+  if (layer instanceof ol.layer.Group) {
+    layer.getLayers().forEach(this.unregisterLayer_, this);
+  }
+
```

Comment:
Thanks @adube 
You are right group should not be listen.
@olivierSemet will fix this and fix his test in an other PR.